modules/packages/Python.chpl

```diff
@@ -1049,20 +1054,74 @@ module Python {
     /*
       Converts an array to a python set
     */
-    // proc toSet(arr): PyObjectPtr throws {
-    // TODO
-    // }
+    proc toSet(arr): PyObjectPtr throws
+      where isArrayType(arr.type) &&
+            arr.rank == 1 && arr.isDefaultRectangular() {
+      var pySet = PySet_New(nil);
+      this.checkException();
+      for i in 0..<arr.size {
+        const idx = arr.domain.orderToIndex(i);
+        PySet_Add(pySet, toPython(arr[idx]));
+        this.checkException();
+      }
+      return pySet;
+    }
```

Comment:
I don't think this overload is called or used?

Reply:
Fair, I added it because List had one as well

Reply:
Since the doesn't get called anywhere (and its matching fromSet), can you add standalone tests of it?